question27/css/style.css

```diff
@@ -0,0 +1,222 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+* {
+    box-sizing: border-box;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+img {
+    max-width: 100%;
+}
+body {
+    height: 100vh;
+    position: relative;
+    background-color: #9e9e9e1c;
+}
+.loading {
+    width: 100px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.wrapper {
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 500px;
+    margin: auto;
+    background: #fff;
+    padding: 35px 5%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translateY(-50%) translateX(-50%);
+    box-shadow: 3px 6px 10px #999;
+}
+h1 {
+    text-align: center;
+    color: #000;
+    font-size: 30px;
+    margin-bottom: 30px;
+    font-family: 'Didot', sans-serif;
+    letter-spacing: 2px;
+}
+h2 {
+    text-align: center;
+    font-size: 28px;
+    margin-bottom: 25px;
+}
+.modal p {
+    font-size: 15px;
+}
+ol ol {
+    list-style: upper-alpha;
+}
+li {
+    margin-bottom: 20px;
+    list-style: none;
+    height: 75px;
+}
+.submit_button {
+    width: 100%;
+    background: #000;
+    font-size: 1rem;
+    line-height: 1.2;
+    font-weight: 300;
+    height: 60px;
+    color: #fff;
+    border-radius: 30px;
+    cursor: pointer;
+}
+.submit_button:disabled {
+    background: #ccc;
+    cursor: not-allowed;
+}
```

Comment:
IMO
I will share what I have learned so far from the review.
Normally, after you type everything in the input, you have to click somewhere else for the validation to go through, but with `pointer-events`, you can press the submit button directly after you type :)

```suggestion
.submit_button {
    width: 100%;
    background: #000;
    font-size: 1rem;
    line-height: 1.2;
    font-weight: 300;
    height: 60px;
    color: #fff;
    border-radius: 30px;
    cursor: pointer;
    pointer-events: auto; //here
}
.submit_button:disabled {
    background: #ccc;
    cursor: not-allowed;
    pointer-events: none; //here
}
```

Reply:
Thank you for your review! This is very convenient! Thanks for letting me know! 😄 

fixed commit : https://github.com/csakyo/homework/pull/41/commits/456f01c744fd68b9bd4eb394b0e403c4b90d0c84